src/components/dialogs/network-modifications/generator/creation/generator-creation-dialog.tsx

```diff
@@ -325,6 +344,7 @@ const GeneratorCreationDialog = ({
         <CustomFormProvider validationSchema={formSchema} {...formMethods}>
             <ModificationDialog
                 fullWidth
+                onClose={clear}
```

Comment:
We don't need this ?

Reply:
Ok it comes from ModificationDialog.propTypes. Maybe we can remove the isRequired from onClose. It seems useless